apps/heureka-next/src/index.scss

```diff
@@ -4,8 +4,11 @@
 body {
   margin: 0;
   overflow: hidden;
-
+  display: flex;
```

Comment:
- Why do you need this?
- I just also have seen that this required part is missing: https://github.com/cloudoperators/juno/blob/main/apps/heureka/index.html#L16-L30
This is required to load the application correctly in embedded mode

Reply:
good to know that putting styles in this way will not work in embedded.